src/harness/reference_models/propagation/ehata/ehata.py

```diff
@@ -86,18 +86,18 @@ def CbsdEffectiveHeights(height_cbsd, its_elev):
   """
 
   npts = int(its_elev[0])
-  xi = its_elev[1] / 1000   # step size of the profile points, in km
+  xi = its_elev[1] / 1000.   # step size of the profile points, in km
```

Comment:
This is good.

Reply:
Most likely this has no effect as its_elev[1] tend to be float, but just in case.